packages/create/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "create-keystone-app",
```

Comment:
If we name the package `@keystone-6/create`, users will be able to just do

```shell
npm create @keystone-6
```

Reply:
Interesting, don't mind that! We'll need to update the documentation though